worker.go

```diff
@@ -321,8 +336,26 @@ func getAllLoadBalancers(certTTL time.Duration, stacks []*aws.Stack) []*loadBala
 	return loadBalancers
 }
 
+func getClusterLocal(loadBalancers []*loadBalancer) *loadBalancer {
+	for _, lb := range loadBalancers {
+		if lb.clusterLocal {
+			return lb
+		}
+	}
+
+	return &loadBalancer{
+		clusterLocal: true,
+	}
+}
+
 func matchIngressesToLoadBalancers(loadBalancers []*loadBalancer, certs CertificatesFinder, certsPerALB int, ingresses []*kubernetes.Ingress) []*loadBalancer {
 	for _, ingress := range ingresses {
+		if ingress.ClusterLocal {
+			lb := getClusterLocal(loadBalancers)
```

Comment:
Since the `loadBalancers` we pass to `matchIngressesToLoadBalancers` are only build based on current stacks, it will never contain a `ClusterLocal` LB. I think it's maybe simpler to just define a `ClusterLocal` `loadBalancer` before the loop and then just add ingresses to that instead of the `getClusterLocal` function which will go through all load balancers and return a new one always anyway.

Reply:
changed as suggested